package.json

```diff
@@ -45,19 +45,26 @@
     "dist"
   ],
   "peerDependencies": {
+    "@ohif/i18n": "^0.0.1",
+    "i18next": "^17.0.3",
     "react": "^16.8.6",
-    "react-dom": "^16.8.6"
+    "react-dom": "^16.8.6",
+    "react-i18next": "^10.11.0"
   },
   "dependencies": {
     "@babel/runtime": "7.2.0",
+    "@ohif/i18n": "0.0.1",
```

Comment:
Why is is in both peerDependencies AND dependencies?

Reply:
When we’re running react-viewerbase locally and outside of OHIF Viewer, it needs the @ohif/i18n as a dependency to run the docs.

Rollup doesn’t include the package when it’s running in the production version.